index.js

```diff
@@ -115,12 +114,51 @@ var isPercyEnabled = true;
 module.exports = {
   name: 'ember-percy',
 
+  _clientInfo: function() {
+    if(!this._clientInfoCache) {
+      var version = require('./package.json').version;
```

Comment:
I've checked with `npm pack` and the `package.json` file does indeed end up in the same directory as this `index.js` file. So we should be safe on this one.